README.md

```diff
@@ -1,2 +1,171 @@
 # megafuel-go-sdk
-Golang SDK for the eoa-paymaster clients
+
+This Golang SDK is thin wrapper of MegaFuel clients, offering a streamlined interface to interact with the [MegaFuel](https://docs.nodereal.io/docs/megafuel-overview).
+
+## Quick Start
+
+1. Install dependency
+
+ ```shell
+ $ go get -u github.com/nodereal/megafuel-go-sdk
+ ```
+
+2. Example
+
+```go
+package main
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/gofrs/uuid"
+	"github.com/node-real/megafuel-go-sdk/pkg/paymasterclient"
+	"github.com/node-real/megafuel-go-sdk/pkg/sponsorclient"
+)
+
+const PAYMASTER_URL = "https://bsc-megafuel-testnet.nodereal.io"
+const CHAIN_URL = "https://data-seed-prebsc-2-s1.binance.org:8545/"
+const SPONSOR_URL = "https://open-platform.nodereal.io/<api-key>/megafuel-testnet"
+
+const POLICY_UUID = "a2381160-xxxx-xxxx-xxxxceca86556834"
+const TOKEN_CONTRACT_ADDRESS = "0xeD2.....12Ee"
+const RECIPIENT_ADDRESS = "0x8e9......3EA2"
+const YOUR_PRIVATE_KEY = "69......929"
+
+func createERC20TransferData(to common.Address, amount *big.Int) ([]byte, error) {
```

Comment:
example too complicated, it can be just send BNB.

Reply:
fixed